### ISSUE TITLE
occ commands are not exiting with error status after symfony/console 3.4.16 update

### ISSUE THREAD

**Issue by phil-davis** [owncloud/core#32978]
See issue raised with the symfony project: https://github.com/symfony/symfony/issues/28695

### Steps to reproduce
1) have ownCloud installed with ``symfony/console`` 3.4.15 e.g.:
```
git checkout v10.0.10
```

2) run some symfony console ``occ`` command in a way that gives an error, e.g. an example ownCloud console command:
```
php occ user:add aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
Enter password: 
Confirm password: 

In Manager.php line 355:
                                                     
  The username can not be longer than 64 characters  

user:add [--password-from-env] [--display-name [DISPLAY-NAME]] [--email [EMAIL]] [-g|--group [GROUP]] [--] <uid>

$ echo $?
1
```
3) update to ``symfony/console`` 3.4.16
```
composer update symfony/console
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 0 installs, 1 update, 0 removals
  - Updating symfony/console (v3.4.15 => v3.4.16): Loading from cache
Writing lock file
Generating optimized autoload files
```
4) repeat the console command:
```
php occ user:add aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
Enter password: 
Confirm password: 

In Manager.php line 355:
                                                     
  The username can not be longer than 64 characters  

user:add [--password-from-env] [--display-name [DISPLAY-NAME]] [--email [EMAIL]] [-g|--group [GROUP]] [--] <uid>

$ echo $?
0
```

### Expected behaviour
Exit status should be 1

### Actual behaviour
Exit status is 0

### Server configuration
**Operating system**:
Ubuntu 18.04

**Web server:**
Development environment

**Database:**
mySQL

**PHP version:**
5.6.38 7.1.22 and 7.2.10 all show the same problem.

**ownCloud version:** (see ownCloud admin page)
core ``stable10`` with recent ``symfony/console`` bump to 3.4.16

**Comment by phil-davis** [comment]
This was noticed because password policy tests started failing. They do ``occ`` password change commands and check the exit status.

It will be good to have a range of core tests that include checking ``occ`` command status. That is actually planned to be started in the current QA sprint.

**Comment by ownclouders** [comment]
[GitMate.io](https://gitmate.io) thinks possibly related issues are https://github.com/owncloud/core/issues/26902 (error), https://github.com/owncloud/core/issues/28840 (-Error-), https://github.com/owncloud/core/pull/30380 ([stable10] Bump symfony/translation from 3.2.4 to 3.3.16), https://github.com/owncloud/core/issues/28265 (Hi,), and https://github.com/owncloud/core/issues/29158 (Dear Support, ).

**Comment by phil-davis** [comment]
Fix by symfony PR https://github.com/symfony/symfony/pull/28664
So should be in 3.4.17 "real soon now" (tm)

**Comment by phil-davis** [comment]
They released symfony 3.4.17 just now, including the fix for the regression. I have tested the ``occ`` command exit status and it works again.
